x11-wm/qtile/qtile-0.22.1.ebuild

```diff
@@ -0,0 +1,73 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_USE_PEP517=setuptools
```

Comment:
Not sure i understand what you mean but i tried to follow [this ebuild](https://github.com/gentoo/gentoo/tree/master/x11-base/xwayland).

Reply:
@thesamesam not sure that live ebuild is needed at all, releases are not so frequent, otherwise feel free to merge

Reply:
I might be one of the few people that uses Qtile on gentoo.

The live ebuild is me being greedy, installing Qtile from git is giving me a lot of trouble and the ebuild seems to solve these problems.

Feel free to remove it since i use the ebuild from my testing overlay.

But of course the version bump is necessary because the old one have terrible Wayland support, and well it's a new release.